api/src/main/java/jakarta/persistence/Query.java

```diff
@@ -558,8 +558,9 @@ Query setParameter(int position, Date value,
     CacheStoreMode getCacheStoreMode();
 
     /**
-     * Set the query timeout.
-     * @param timeout the timeout, or null to indicate no timeout
+     * Set the query timeout, in milliseconds. This is a hint.
+     * @param timeout the timeout, in milliseconds, or null to
+     *                indicate no timeout
```

Comment:
same change is needed in TypedQuery and StoredProcedureQuery

Reply:
Done.